components/atom/progressBar/src/index.js

```diff
@@ -0,0 +1,53 @@
+import React, {PureComponent} from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+const BASE_CLASS = 'sui-AtomProgressBar'
+const CLASS_INDICATOR = 'sui-AtomProgressBar-indicator'
+const CLASS_CONTAINER_BAR = 'sui-AtomProgressBar-container'
+const CLASS_BAR = 'sui-AtomProgressBar-bar'
+
+class AtomProgressBar extends PureComponent {
+  renderIndicator(indicatorBottom) {
```

Comment:
Usually, I would like more a component itself than a render method.

Reply:
Also, IMHO it's a bit weird the implementation of this method. As the indicatorBottom is retrieved from param but still is accessing from props the rest of props. I would or use only named params or only use props.

Reply:
👍